src/sirinternal.c

```diff
@@ -33,7 +33,11 @@
 #include "sir/mutex.h"
 
 #if defined(__WIN__)
-# pragma comment(lib, "ws2_32.lib")
```

Comment:
This is making me want to pull all of these comment(lib) pragmas into one file... say, sirplatform.c...